packages/config/src/config/prefix.ts

```diff
@@ -4,4 +4,5 @@ export enum ConfigPrefix {
   Provider = 'pv',
   ImageryRule = 'ir',
   Style = 'st',
+  Job = 'job',
```

Comment:
have three letters here will break other logic, there are quite a few places where it expects the prefix to be three lettters total `pv_`